pkg/envoy/cds.go

```diff
@@ -1,14 +1,13 @@
 package envoy
 
 import (
-	"context"
 	"fmt"
 
+	"github.com/openservicemesh/osm-health/pkg/kuberneteshelper"
```

Comment:
@SanyaKochhar The `/openservicemesh/` packages should be grouped in the last section of imports